composables/useListingCartConfig.ts

```diff
@@ -0,0 +1,15 @@
+export const displayOnPages = [
```

Comment:
why create a new composable instead of puting the code inside cart `store`?

Reply:
It made sense to me to have config out of the store

Let me know if you disagree and I'll change that